client/cli/Cargo.toml

```diff
@@ -25,7 +25,7 @@ libp2p = "0.30.1"
 parity-scale-codec = "1.3.0"
 hex = "0.4.2"
 rand = "0.7.3"
-bip39 = "0.6.0-beta.1"
+tiny-bip39 = "0.8.0"
```

Comment:
If this change is the only required one, it means we don't require this dependency?

Reply:
no, actually the library name of `tiny-bip39` is `bip39` :P, so we don't need to modify the rust code.

Reply:
For context: `tiny-bip39` is a fork of `bip39` by @maciejhirsz that includes some good cleanup, including some secret-zeroing that would be good to have. The original crate is maintained but slow moving. There are signs of a `bip39 1.0` coming at some point.
As @koushiro points out, the two crates are API compatible, so switching back should be fairly easy.